### PR TITLE
Add missing doc-string for `blend` API

### PR DESCRIPTION
**Pull request by Nic-Ma**
### Description
I am working on the visualization tutorial https://github.com/Project-MONAI/tutorials/pull/448 and found that there is no doc-string for `blend` args and it's not easy to get the meaning of these args. So this PR enhanced the doc-string for it.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/visualize/utils.py [hunk 1]
```diff
@@ -131,9 +131,22 @@ def matshow3d(
...
+    """
+    Blend a image and a label. Both should have the shape CHW[D].
     The image may have C==1 or 3 channels (greyscale or RGB).
-    The label is expected to have C==1."""
+    The label is expected to have C==1.
+
+    Args:
+        image: the input image to blend with label data.
+        label: the input label to blend with image data.
+        alpha: when blending image and label, `alpha` is weight for the image region mapping to `label != 0`,
```
Comment (yiheng-wang-nv) [inline monai/visualize/utils.py:142]: `alpha` is the weight

Comment (yiheng-wang-nv) [reply monai/visualize/utils.py:142]: `1 - alpha` is the weight

File: monai/visualize/utils.py [hunk 2]
```diff
@@ -131,9 +131,22 @@ def matshow3d(
...
+            and `1 - alpha` is weight for the label region that `label != 0`.
+        cmap: specify colormap in the matplotlib, for more details, please refer to:
+            https://matplotlib.org/2.0.2/users/colormaps.html.
+        rescale_arrays: whether to rescale the array to [0, 1] first.
```
Comment (yiheng-wang-nv) [inline monai/visualize/utils.py:146]: Defaults to `True` (also for cmap, alpha)

File: monai/visualize/utils.py (hunk 1 above)
Comment (Nic-Ma) [reply monai/visualize/utils.py]: Sure, updated in the latest commit.

Thanks.

File: monai/visualize/utils.py (hunk 2 above)
Comment (Nic-Ma) [reply monai/visualize/utils.py]: Sure, updated in the latest commit.

Thanks.

**Review by yiheng-wang-nv — APPROVED** [review]

**Pull request merged**